File: Clase02/ProyectoClases/Persona.vb
Public Class Persona

    Private id As Integer
    Private paterno As String
    Private materno As String
    Private nombres As String
    Private nombreCompleto As String
    Private edad As Integer
    Private fechaNacimiento As DateTime




    Public Sub nombreCompletoPersona()
        nombreCompleto = nombres + materno + paterno
    End Sub


    Public Sub New(
                  id As Integer,
                  paterno As String,
                  materno As String,
                  nombres As String,
                  nombreCompleto As String,
                  edad As Integer,
                  fechaNacimiento As DateTime)
        id = id
        paterno = paterno
        materno = materno
        nombres = nombres
        nombreCompleto = nombreCompleto
        edad = edad
        fechaNacimiento = fechaNacimiento
    End Sub

    Public Sub New(
                  id As Integer,
                  paterno As String,
                  materno As String,
                  nombres As String,
                  fechaNacimiento As DateTime)
        id = id
        paterno = paterno
        materno = materno
        nombres = nombres
        fechaNacimiento = fechaNacimiento
    End Sub

End Class

File: clase02/ConsoleApp2/Persona.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApp2
{
    class Persona
    {
        public int id { get; set; }
        public String apellidos { get; set; }
        public String nombres { get; set; }
        public String nombreCompleto { get; set; }
        public int edad { get; set; }
        public DateTime fechaNacimiento { get; set; }

        public Persona(int id, string apellidos, string nombres, DateTime fechaNacimiento)
        {
            this.id = id;
            this.apellidos = apellidos;
            this.nombres = nombres;
            this.fechaNacimiento = fechaNacimiento;
        }
    }
}
